:sparkles: Add firebase on the project (#35)
Zenika/deadoralive: README.md

## Install

```sh
yarn
```

### Firebase configuration

> (firebase configuration is optionnal, if there are no config, the score isn't stored)

Reviewer: ```suggestion
Firebase configuration is optional, if unavailable, player's data won't be stored.
```

> * Create firebase project.

Reviewer: ```suggestion
* Create firebase project
```
